src/TreeToTS/functions/new/models.ts

```diff
@@ -65,21 +65,35 @@ export type OperationOptions = {
 
 export type ScalarCoder = Record<string, (s: unknown) => string>;
 
+import { GraphQLError, type GraphQLErrorOptions } from 'graphql'; // keep
```

Comment:
Pros:
- No external dependencies
- Backports the feature to users of graphql@<=15

Cons:
- Feels flaky
- [We will need to bundle a lot of things](https://github.com/graphql/graphql-js/blob/main/src/error/GraphQLError.ts)

Alternative: create a lite version of GraphQLError, but it will be incompatible with its `graphql-js#GraphQLError` counterpart, and we will need to export it as `GraphQLError` which is a breaking change.